lib/utilities/strings.rb

```diff
@@ -58,9 +59,18 @@ def self.termify(string)
   # @return [Boolean]
   #   whether the string is an integer (positive or negative)
   # see http://stackoverflow.com/questions/1235863/test-if-a-string-is-basically-an-integer-in-quotes-using-ruby
+  # Note:  Might checkout CSV::Converters constants to see how they handle this
   def self.is_i?(string)
     /\A[-+]?\d+\z/ === string
   end
 
+  # @return [String, param]
+  #   the goal is to sanitizie an individual string such that it is usable in *TAB* delimited, UTF-8, column.  See Download
+  #   TODO: Likely need to handle quotes
+  def self.sanitize_for_csv(string)
+    return string if string.blank?
+    string.to_s.gsub(/\n/, '\n').gsub(/\t/, '\t')
```

Comment:
Consumers won't necessarily understand this C-like character escape sequences. Typically you would replace this by empty string and a space respectively. However, it looks like CSV tolerates new lines and tabs as demonstrated by the following script:
```
hernan@hernan-pc:~$ irb
2.2.1 :001 > require 'csv'
 => true 
2.2.1 :002 > csv_string = CSV.generate do |csv|
2.2.1 :003 >       csv << ["Multi\nLine", "name:\tHernán", "Computers are \"silly\" devices"]
2.2.1 :004?>   end
 => "\"Multi\nLine\",name:\tHernán,\"Computers are \"\"silly\"\" devices\"\n" 
2.2.1 :005 > puts csv_string
"Multi
Line",name:	Hernán,"Computers are ""silly"" devices"
 => nil 
```
But yet, to be extra safe I would suggest replacing \t with " " and the rest of the control characters with "". Quotes are escaped by duplicating them.